src/server/dflycmd.cc

```diff
@@ -540,6 +540,9 @@ void DflyCmd::TakeOver(CmdArgList args, RedisReplyBuilder* rb, ConnectionContext
 
   // For non-cluster mode we shutdown
   if (detail::cluster_mode != detail::ClusterMode::kRealCluster) {
+    // we don't have flush() mechanism for ReplyBuilder, so we wait a bit to be sure that reply was
```

Comment:
1s is not a bit.  have you checked it helps or is it a guess?

Reply:
it's not critical but if a replica got the reply, it will break the connection itself. In that case `Connection::OnBreakCb` will update `cc_->conn_closing = true;` which can serve as a confirmation to proceed further. So I would actually here do something like:
```
for (unsigned i = 0; i < 50; ++i) {
  if (cntx->conn_closing) {
    VLOG(1) << "Take over confirmed";
     break;
  }
  SleepFor(20ms);
}
```
or something like that. and then manually check that it helps on long-latency connection where before that it failed.